File: src/main/java/popa/catalin/lab5/ex1_producer_consumer/MailService.java
package popa.catalin.lab5.ex1_producer_consumer;

public class MailService {

    public void sendMail(String msg) throws InterruptedException {
        System.out.printf("Sending email {%s} \n", msg);
        Thread.sleep(1000);
        System.out.println("Email sent.");
    }
}

File: LabsSimonaBurlacu232/Lab5/ProducerConsumer/EmailSender/EmailReceiver/MailService.cs
using System;
using System.Threading;

namespace EmailReceiver
{
    public class MailService
    {
        public void SendEmail(string message)
        {
            Console.WriteLine("Sending message " + message);
            Thread.Sleep(1000);
            Console.WriteLine("Email sent");
        }
    }
}
